### ISSUE TITLE
NTR: lncRNA (N6-adenosine)-methyltransferase activity

### ISSUE THREAD

**Issue by Antonialock** [geneontology/go-ontology#27200]
Please provide as much information as you can: 

* **Suggested term label:**
lncRNA (N6-adenosine)-methyltransferase activity

* **Definition (free text)**
Catalysis of the reaction:  S-adenosyl-L-homocysteine + lncRNA containing N6-methyladenine. 

* **Reference, in format PMID:#######**
(REQUIRED)
 34980213

* **Gene product name and ID to be annotated to this term**

Q9H8H3 TMT1a

* **Parent term(s)**

GO:0008173
* **Children terms (if applicable)** Should any existing terms that should be moved underneath this new proposed term?

* **Synonyms (please specify, EXACT, BROAD, NARROW or RELATED)**

* **Cross-references**
* For enzymes, please provide RHEA and/or EC numbers. 
* Can also provide MetaCyc, KEGG, Wikipedia, and other links.

* **Any other information**

**Comment by ValWood** [comment]
Do we need an lncRNA-specific term? It seems we need a general

RNA (adenine-N6-)-methyltransferase

And that could be used with the specific lncRNA as input?

and move other terms underneath it
GO:0008988   https://www.ebi.ac.uk/QuickGO/services/ontology/go/terms/GO:0008988/complete)

rRNA (adenine-N6-)-methyltransferase activity
etc

**Comment by pgaudet** [comment]
I agree, if we can avoid naming the substrate here, that would be nice. 

I am aware that we have tRNA and other substrates, in part to follow RHEA and to name specific residue. 

(I see that the gene @Antonialock wants to annotate, Q9H8H3 TMT1a, already has a RHEA that describes mRNA: 
https://www.rhea-db.org/rhea/55584

Maybe we can make the GO term broader and make the RHEA a narrow match ?

**Comment by raymond91125** [comment]
@ai4c-agent please make a proposal for resolving this issue. Is creating a new term RNA (adenine-N6)-methyltransferase activity a good idea and how to organize the ontology with it?

**Comment by raymond91125** [comment]
@ai4c-agent please check if this term should be moved under RNA (adenine-N6)-methyltransferase activity? GO:0120048 U6 snRNA (adenine(43)-N6)-methyltransferase activity. Also, find specific PMID evidence for RNA (adenine-N6)-methyltransferase activity, not your own.

**Comment by raymond91125** [comment]
@ai4c-agent please go ahead with 
Create the new GO:777xxxx RNA (adenine-N6)-methyltransferase activity with the two-PMID xref
Reparent GO:0001734, GO:0008988, GO:0106348, and GO:0120048 (four children

**Comment by raymond91125** [comment]
id: GO:7770107
name: RNA (adenine-N6)-methyltransferase activity
namespace: molecular_function
def: "Catalysis of the reaction: an adenosine in RNA + S-adenosyl-L-methionine = an N(6)-methyladenosine in RNA + S-adenosyl-L-homocysteine + H+." [PMID:34023900, PMID:36736310]
synonym: "RNA m6A methyltransferase activity" EXACT []
synonym: "RNA (N6-adenosine)-methyltransferase activity" EXACT []
is_a: GO:0008170 ! N-methyltransferase activity
is_a: GO:0008173 ! RNA methyltransferase activity
property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27200" xsd:anyURI
created_by: ai4c-agent
creation_date: 2026-08-07T23:44:01Z
